_config.yml

```diff
@@ -2,16 +2,16 @@
 # Values for this lesson.
 #------------------------------------------------------------
 
-# Which carpentry is this ("swc", "dc", "lc", or "cwl")?
-carpentry: "cwl"
+# Which carpentry is this ("swc", "dc", or "lc")?
+carpentry: "swc"
```

Comment:
I made my own prefix so it'll be easier to merge from the template repo. What do you think?